tests/integration-e2e/test_cli_opts.py

```diff
@@ -165,3 +165,24 @@ def test_cli_output_path():
     assert not (test_folder_path / simconfig_output_path).is_dir(), \
            f"Directory '{simconfig_output_path}' should NOT exist."
     assert (test_folder_path / output_path).is_dir(), f"Directory '{output_path}' not found."
+
+def test_cli_report_buff_size():
+    test_folder = tempfile.TemporaryDirectory("cli-test-report-buff-size")  # auto removed
```

Comment:
please don't use `tempfile`; use the `tmp_path` fixture; even better, there is the `create_tmp_simulation_config_file` fixture that does this for you, I think

Reply:
I'll keep this construct for now to be consistent with the other existing tests. Maybe we can do a proper switch in the future

Reply:
ah right, good point - forgot that was the dependency.

Reply:
totally agree. You can check `unit/test_ringcells.py` for inspiration. Or this:

```
@pytest.mark.parametrize("create_tmp_simulation_config_file", [
    {
        "simconfig_fixture": "ringtest_baseconfig",
        "extra_config": {
            "target_simulator": "CORENEURON",
            "node_set": "Mosaic"
        }
    }
], indirect=True)
def test_coreneuron(create_tmp_simulation_config_file):
```

Reply:
Latest commit has adapted to the structure listed